### PR TITLE
feat: add max width translations

### PR DESCRIPTION
**Pull request by pyphilia**

### CODE REVIEW

File: src/langs/fr/maxWidth.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "sm": "Petit",
+  "md": "Moyen",
+  "lg": "Grand",
+  "xl": "Extrèmement Grand"
```
Comment (spaenleh) [inline src/langs/fr/maxWidth.json:5]:
```suggestion
  "xl": "Extrêmement Grand"
```

**Review by spaenleh — APPROVED** [review]
Is the `xl` supposed to be maximum width ?

**Comment by pyphilia** [comment]
@spaenleh  Yes!

**Comment by spaenleh** [comment]
okay, well in the docs it is set as `1536px` bit I will take it 😄

**Pull request merged**